cpp/src/sampling/nbr_sampling.cu

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Better rename this to `nbr_sampling_mg.cu` to follow our convention.

Reply:
Will do.